compose/backend.yml

```diff
@@ -2,6 +2,35 @@
 version: '3'
 
 services:
+
+  annotation-api:
```

Comment:
the backends file should contain only the core CCD backends. All side services like media viewer should be defined in a separate compose file.
This allows to start the core CCD with little mem and CPU resources. 
Whoever needs MV can then enable it explicitly 
Please have a chat with me on how to do this I will help

Reply:
Done. Please review and approve.